deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -245,6 +245,21 @@ def _set_elasticsearch_kubernetes_resources():
     if not has_kube_resource:
         run('kubectl apply -f deploy/kubernetes/elasticsearch/kubernetes-elasticsearch-all-in-one.yaml')
 
+
+def deploy_linkerd(settings):
+    print_separator('linkerd')
```

Comment:
you should probably add this to the `DEPLOYMENT_TARGETS` config at the top of the file